packages/common/src/hardhat/tasks/finder.js

```diff
@@ -0,0 +1,83 @@
+const { task } = require("hardhat/config");
+const { interfaceName } = require("@uma/common");
+
+task("setup-finder", "Points Finder to DVM system contracts")
+  .addFlag("registry", "Use if you want to set Registry")
+  .addFlag("generichandler", "Use if you want to set GenericHandler")
+  .addFlag("bridge", "Use if you want to set Bridge")
+  .addFlag("identifierwhitelist", "Use if you want to set IdentifierWhitelist")
+  .addFlag("addresswhitelist", "Use if you want to set AddressWhitelist")
+  .addFlag("financialcontractsadmin", "Use if you want to set FinancialContractsAdmin")
+  .addFlag("optimisticoracle", "Use if you want to set OptimisticOracle")
+  .addFlag("store", "Use if you want to set Store")
+  .addFlag("mockoracle", "Use if you want to set MockOracle as the Oracle")
+  .addFlag("sinkoracle", "Use if you want to set SinkOracle as the Oracle")
+  .addFlag("prod", "Configure production setup in Finder")
```

Comment:
You can set up the Finder really easily now via `yarn hardhat setup-finder --prod --sinkoracle --network matic` for example.